english/revolution/revolution.md

```diff
@@ -6,4 +6,6 @@ time for the people's voices to be heard. Our choices are:
 
 [Go to war with Clinton.](choice_1/Clinton.md)
 
-[Go to an economic crash with Trump.](choice_2/Trump.md)
\ No newline at end of file
+[Go to an economic crash with Trump.](choice_2/Trump.md)
+
+ Want to see [Egypt revolution](english/Egypt-the-4000-year-of-civilization/25th-january-revolution/revolution.md) to get prepared?
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.